lang/yabasic/Portfile

```diff
@@ -1,25 +1,32 @@
-PortSystem 1.0
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
 
-name			yabasic
-version			2.763
-revision                1
-categories		lang
-platforms		darwin
-license			{Artistic-1 GPL-2}
-maintainers		nomaintainer
-description		yet another basic language
-long_description	Yabasic implements the most common and simple elements \
-				of the basic language. It comes with goto/gosub, with \
-				various loops, with user defined subroutines and Libraries. \
-				Yabasic does monochrome line graphics and printing.
+name                yabasic
+version             2.86.7
+revision            0
+categories          lang
+platforms           darwin
+license             {Artistic-1 GPL-2}
+maintainers         nomaintainer
+description         yet another basic language
+long_description    Yabasic implements the most common and simple elements \
+                    of the basic language. It comes with goto/gosub, with \
+                    various loops, with user defined subroutines and Libraries. \
+                    Yabasic does monochrome line graphics and printing.
 
-homepage		http://www.yabasic.de/
-master_sites	${homepage}/download/
-checksums		md5 98f0cb59db973b89753abcc24b2c5ec2
+homepage            http://www.yabasic.de/
+master_sites        ${homepage}/download/
+checksums           md5 061602367793e122e8dfb26947853bcf \
```

Comment:
You can remove the md5 checksum, we no longer require this.

Reply:
Done.